examples/with-storybook/stories/button.stories.js

```diff
@@ -1,17 +1,25 @@
 import React from 'react'
-import { action } from '@storybook/addon-actions'
 import { Button } from '@storybook/react/demo'
 
-export default { title: 'Button' }
+export default {
+  title: 'Button',
+  argTypes: { onClick: { action: 'clicked' } },
+}
 
-export const withText = () => (
-  <Button onClick={action('clicked')}>Hello Button</Button>
-)
+const TemplateWithText = (args) => <Button {...args}>Hello Button</Button>
```

Comment:
`args` would be more like `arguments`, so I prefer using `props` instead.

Reply:
In storybook, the props is called args:

https://storybook.js.org/docs/react/writing-stories/args